packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -1072,6 +1072,107 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /agreements/{agreementId}/consumerDocuments/{documentId}:
+    parameters:
+      - name: agreementId
+        in: path
+        description: The ID of the Agreement
+        required: true
+        schema:
+          type: string
+          format: uuid
+      - name: documentId
+        in: path
+        description: The ID of the Document to retrieve or delete
+        required: true
+        schema:
+          type: string
+          format: uuid
+    get:
+      tags:
+        - agreements
+      summary: Retrieve an Agreement Consumer Document
+      description: Retrieve a specific Consumer Document for the specified Agreement
+      operationId: getAgreementConsumerDocument
```

`retrive` is perfectly fine, this minor suggestion is just to make clear the different operation

```suggestion
      summary: Download an Agreement Consumer Document
      description: Download a specific Consumer Document for the specified Agreement
      operationId: downloadAgreementConsumerDocument
```